src/templates/FoodDetails.tsx

```diff
@@ -171,8 +172,15 @@ const FoodDetails: React.FC<Props> = ({ pageContext: { recommendedDailyIntakes,
     setDefaultSelectedAgeForGender(selectedLifeStage, setSelectedAge);
   }, [selectedLifeStage, setSelectedAge]);
 
+  console.log(slug);
+
   return (
-    <div>
+    <div
+      id={`ff-page-${slug}`}
+      data-json={JSON.stringify(nutritionFacts)}
+      age-json={JSON.stringify(ageRanges)}
+      selected-gender={selectedLifeStage}
```

Comment:
Remove `selected-gender`, not adding much value

Reply:
Done